libs/designer/src/lib/ui/mcp/parameters/EditOperation.tsx

```diff
@@ -282,10 +304,19 @@ export const EditOperation = () => {
     (param: ParameterInfo, isConditional = false) => {
```

Comment:
nit: a little bit of cleanup, but can we moverenderParameterField to its own component so that we don't need to pass styles, intl into dependencies